modules/common/react/lib/theming/CanvasProvider.tsx

```diff
@@ -1,8 +1,8 @@
 import * as React from 'react';
 import {ThemeProvider} from 'emotion-theming';
 import {InputProvider} from '@workday/canvas-kit-react-core';
-import {CanvasTheme, ContentDirection} from './theming/types';
-import {defaultCanvasTheme} from './theming/theme';
```

Comment:
`CanvasProvider` is not specific to theming. It should be moved out of the `theming/` folder